microsoft-edge/visual-studio/index.md

```diff
@@ -34,52 +34,49 @@ The simplified workflow allows you to debug JavaScript that run in Microsoft Edg
 
 ### Create a new ASP.NET Core web app  
 
-Open Visual Studio 2019 and choose **Create a new project**.  On the next screen, choose **ASP.NET Core Web app** > **Next**.  
+Open Visual Studio 2019 and select **Create a new project**.  In the search box on the next screen, enter **react**.  Select **ASP.NET Core with React.js** from the list of templates and then **Next**.  
 
-:::image type="complex" source="./media/create-new-project.png" alt-text="Create a new ASP.NET Core Web app" lightbox="./media/create-new-project.png":::
-   Create a new ASP.NET Core Web app  
-:::image-end:::  
+This **React.js** template specifies how to integrate React.js with an ASP.NET Core app.  
 
-Provide a **Project name** for your new project and choose **Create**.  For the purposes of the example, choose **React.js** as the template, and choose **Create**.  The **React.js** template specifies how to integrate React.js with an ASP.NET Core app.  
+:::image type="complex" source="./media/create-new-project.png" alt-text="Create a new ASP.NET Core Web app with React.js" lightbox="./media/create-new-project.png":::
+   Create a new ASP.NET Core Web app with React.js  
+:::image-end:::  
 
 ### Launch Microsoft Edge from Visual Studio  
 
-After you create your project, open `ClientApp/src/components/Counter.js`.  Now, to use Visual Studio to debug JavaScript, choose the dropdown next to the green **Play** button and **IIS Express**.  
+After you create your project, open `ClientApp/src/components/Counter.js`.  Now, to use Visual Studio to debug JavaScript, select the dropdown next to the green **Play** button and **IIS Express**.  
 
 :::image type="complex" source="./media/vs-dropdown.png" alt-text="The dropdown next to the green Play button and IIS Express" lightbox="./media/vs-dropdown.png":::
    The dropdown next to the green **Play** button and **IIS Express**  
 :::image-end:::  
 
-Choose **Script Debugging** > **Enabled**.  
+Select **Script Debugging** > **Enabled**.  
 
 :::image type="complex" source="./media/enable-script-debugging.png" alt-text="Turn on script debugging in Visual Studio" lightbox="./media/enable-script-debugging.png":::
    Turn on script debugging in Visual Studio  
 :::image-end:::  
 
-In the same dropdown, choose **Web Browser** > the preview channel of Microsoft Edge that you want Visual Studio to launch, such as Microsoft Edge Canary, Dev, or Beta.  If you are not already using one of the Microsoft Edge preview channels, navigate to [Download Microsoft Edge Insider Channels][MicrosoftedgeinsiderDownload] to download one.  
+In the same dropdown, select **Web Browser** > the preview channel of Microsoft Edge that you want Visual Studio to launch, such as Microsoft Edge Canary, Dev, or Beta.  If you are not already using one of the Microsoft Edge preview channels, navigate to [Download Microsoft Edge Insider Channels][MicrosoftedgeinsiderDownload] to download one.  
```

Comment:
Elsewhere, change:
you may step through the script line-by-line
to:
you can step through the script line-by-line